src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
The interface name `AreaCalculator` is not ideal since it uses a verb. Consider renaming it to something that represents an entity or a capability, such as `AreaComputable` or `AreaMeasure`, aligning with the Java naming conventions for interfaces which often end with -able or -ible.